docs/developers/build-starrocks/build_starrocks_on_ubuntu.md

```diff
@@ -0,0 +1,52 @@
+### Prerequisite
+
+```
+sudo apt-get install automake binutils-dev bison byacc ccache flex libiberty-dev libtool maven zip python3 python-is-python3 -y
+```
+
+### Compiler
+If the ubuntu version >= 22.04, you can
+```
+sudo apt-get install cmake gcc g++ default-jdk -y
+```
+
+If the ubuntu version < 22.04.
+Check the version of following tools and compilers
+
+##### 1. GCC/G++
+GCC/G++ version must be >= 10.3
+```
+gcc --version
+g++ --version
+```
+Install GCC/G++(https://gcc.gnu.org/releases.html)
+
+##### 2. JDK
+OpenJDK version must be >= 8
+```
+java --version
+```
+Install OpenJdk(https://openjdk.org/install)
+
+##### 3. CMake
+cmake version must be >= 3.20.1
+
+```
+cmake --version
+```
+Install cmake(https://cmake.org/download)
+
+
+### Improve the compile speed
+The default compiling paralleilsim equals to the **CPU Cores / 4**. If you want to improve the compile speed. You can improve the paralleilsim. Suppose you have 32 Cpu cores, want to use 24 cores to compile.
```

Comment:
I think you should provide a normal build method. Otherwise, people will copy the only example `./build.sh -j 24`. Which will cause problems.